tutorials/discovery/c++11/home_automation_subscriber.cxx

```diff
@@ -0,0 +1,60 @@
+//
+// (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+//
+//  RTI grants Licensee a license to use, modify, compile, and create derivative
+//  works of the Software.  Licensee has the right to distribute object form
+//  only for use with RTI products.  The Software is provided "as is", with no
+//  warranty of any type, including any warranty for fitness for any purpose.
+//  RTI is under no obligation to maintain or support the Software.  RTI shall
+//  not be liable for any incidental or consequential damages arising out of the
+//  use or inability to use the software.
+//
+
+#include <iostream>
+#include <thread>
+
+#include "rti/rti.hpp"
+#include "rti/sub/SampleProcessor.hpp"
+#include "home_automation.hpp"
+
+int main(int argc, char **argv)
+{
+    dds::domain::DomainParticipant participant(0);
+    dds::topic::Topic<DeviceStatus> topic(participant, "WindowStatus");
+    dds::sub::DataReader<DeviceStatus> reader(topic);
+
```

Comment:
@alexcamposruiz Do we want to display the received samples too? Or should we keep it simple and just display the new publishers that we discovered?

Reply:
Yes, let's display samples too. This should be an addition to the base publish-subscribe example